src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/PackageDetailsTabViewModel.cs

```diff
@@ -85,13 +83,23 @@ public void Dispose()
             }
             _disposed = true;
             DetailControlModel.PropertyChanged -= DetailControlModel_PropertyChanged;
+
+            foreach (var tab in Tabs)
+            {
+                tab.PropertyChanged -= IsVisible_PropertyChanged;
+            }
         }
 
         private void IsVisible_PropertyChanged(object sender, PropertyChangedEventArgs e)
         {
             if (e.PropertyName == nameof(TitledPageViewModelBase.IsVisible))
             {
-                if (!SelectedTab.IsVisible)
+                if (SelectedTab == null)
+                {
+                    SelectedTab = Tabs.FirstOrDefault(t => t.IsVisible);
+                }
+                else if (SelectedTab == sender
+                    && !SelectedTab.IsVisible)
```

Comment:
These could be combined, especially since they call the same exact code if true.

```suggestion
                if (SelectedTab == null || (SelectedTab == sender && !SelectedTab.IsVisible))
```